Adding Seeders for all tables (fake content for local/dev environments)

MaelCormier-CGI opened the pull request:
- Created Seeders for each table
- Updated DatabaseSeeder files (main db and each module db).
- Updated Readme file

kmarshgov reviewed (changes requested):
Yuikes, you don't want to run these seeders outside of local

        //run the seeders only on local
        if (App::environment('local')) {
            $this->call([
                UserSeeder::class,
                MinistrySeeder::class,
            ]);
        }


For seeds you think the data is needed just like the ones I asked you to add under utils for twp you need to write the seeder differently so it does not duplicate records.
For example:
       Role::updateOrCreate(
            ['name' => 'user'],
            ['description' => 'Regular user role']
        );

So your DatabaseSeeder files will endup with a command to seed on all env and ones only local:
        //essential seeds.
        $this->call(ProductionSeeder::class);

        //for non-production environment.
        if (App::environment('local', 'testing')) {
            $this->call(DevelopmentSeeder::class);
        }

MaelCormier-CGI commented:
Pushed new changes:

- Added safeguard on DB Seeder runner files to run seeders only in local/dev env
- Updated Seeder files to use ->updateOrInsert instead of ->insert, to avoid any duplicates
